index.js

```diff
@@ -0,0 +1,15 @@
+const burger = document.getElementById('fa-bars');
+const hideMenu = document.getElementById('closeIcon');
+const navLink = document.querySelector('.nav-link');
+const navLinks = document.getElementById('desktop-nav');
+
+burger.onclick = function burger() {
+  navLinks.style.right = '0';
+};
+hideMenu.onclick = function hideMenu() {
+  navLinks.style.right = '-100%';
+};
+
+navLink.onclick = function hideMenu() {
```

Comment:
good job @divinecharlotte implementing the open/close menu functionality, but we can improve this portion of code by applying the DRY rule as you can see your 3 functions are doing the same thing so I suggest that you create one function that receives the right attribute as argument and call it as a callback function 
`const hideMenuFunc = (arg) => {
  navLinks.style.right = arg;
};
burger.onclick = () => hideMenuFunc("0");
hideMenu.onclick = () => hideMenuFunc("-100%");
navLink.onclick = () => hideMenuFunc("-100%");`